stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/utils/extensions/Int.kt

```diff
@@ -1,6 +1,7 @@
 package io.getstream.chat.android.ui.utils.extensions
 
 import android.content.res.Resources
+import kotlin.math.ceil
 
-internal fun Int.dpToPx(): Int = dpToPxPrecise().toInt()
+internal fun Int.dpToPx(): Int = ceil(dpToPxPrecise()).toInt()
```

Comment:
Do we _always_ want to round up for some reason, or do we just want to make sure that things like `0.99f` don't get converted to `0` int value?

For the latter, we could use [`roundToInt`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.math/round-to-int.html) which does proper nearest-integer rounding.

Reply:
Yes, we always want, because the old approach will convert 0.99  to 0 and 2.65 to 2. Thanks for `roundToInt` is better than converting to int manually